composer.json

```diff
@@ -21,22 +21,22 @@
     ],
     "require": {
         "php": "^5.5.9 || ^7.0",
-        "symfony/form": "^2.7 || ^3.0",
-        "symfony/framework-bundle": "^2.7 || ^3.0",
-        "symfony/security-bundle": "^2.7 || ^3.0",
-        "symfony/twig-bundle": "^2.7 || ^3.0",
+        "symfony/form": "^2.7 || ^3.0 || ^4.0",
+        "symfony/framework-bundle": "^2.7 || ^3.0 || ^4.0",
+        "symfony/security-bundle": "^2.7 || ^3.0 || ^4.0",
+        "symfony/twig-bundle": "^2.7 || ^3.0 || ^4.0",
         "twig/twig": "^1.28 || ^2.0",
-        "symfony/templating": "^2.7 || ^3.0",
+        "symfony/templating": "^2.7 || ^3.0 || ^4.0",
         "paragonie/random_compat": "^1 || ^2"
     },
     "require-dev": {
         "doctrine/doctrine-bundle": "^1.3",
         "friendsofphp/php-cs-fixer": "^1.11",
         "swiftmailer/swiftmailer": "^4.3 || ^5.0 || ^6.0",
-        "symfony/console": "^2.7 || ^3.0",
-        "symfony/phpunit-bridge": "^2.7 || ^3.0",
-        "symfony/validator": "^2.7 || ^3.0",
-        "symfony/yaml": "^2.7 || ^3.0",
+        "symfony/console": "^2.7 || ^3.0 || ^4.0",
+        "symfony/phpunit-bridge": "^2.7 || ^3.0 || ^4.0",
+        "symfony/validator": "^2.7 || ^3.0 || ^4.0",
+        "symfony/yaml": "^2.7 || ^3.0 || ^4.0",
         "phpunit/phpunit": "~4.8.35|~5.4.3"
```

Comment:
Do you really want to exclude 5.5+? The constraint should probably be `^4.8.35|^5.4.3` (for 4.x it actually doesn't really matter as there is no 4.9 release).

Reply:
Probably not relevant already? Looks like Ryan just reverted it to the original line

Reply:
It's surely not relevant to this PR. We can address that elsewhere.

Reply:
I've proposed the fix in #2647